src/reuse/modules/util/file.ts

```diff
@@ -45,6 +45,43 @@ export class File {
       throw new Error(`Function 'upload' failed': ${error}`);
     }
   }
+  
+  /**
+   * @function uploadWebGui
+   * @memberOf util.file
+   * @description Uploads all the file/s by the paths given in the Array for SAP WebGUI apps.
+   * @param {String[]} files - Array with path/s of file/s to be uploaded.
+   * @param {String | Object} selector - Custom selector of the input element
+   * @example await util.file.uploadWebGui(["path/to/text1.txt"], "INPUT[title='External file name']");
+   */
+  async uploadWebGui(files: Array<string>, selector: any) {
+    const vl = this.vlf.initLog(this.uploadWebGui);
+    try {
+      if (typeof selector === "string") {
+        const elem = await nonUi5.element.getByCss(selector);
+        await nonUi5.userInteraction.click(elem);
+      } else if (typeof selector === "object") {
+        await ui5.userInteraction.click(selector);
```

Comment:
do we need to support the ui5 version in the webgui reuse?

Reply:
what do you mean by that?

Reply:
fixed, changed selector type to string. The input element won't be a ui5 element.